src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/TseAdminHelper.java

```diff
@@ -25,7 +26,9 @@ public static DynamicFixedListType populateSelectApplicationAdminDropdown(CaseDa
         }
 
         return DynamicFixedListType.from(caseData.getGenericTseApplicationCollection().stream()
-            .filter(r -> r.getValue().getStatus() != null && !CLOSED_STATE.equals(r.getValue().getStatus()))
+            .filter(r -> r.getValue().getStatus() != null
+                && !CLOSED_STATE.equals(r.getValue().getStatus())
```

Comment:
What statuses are you trying to get? Would it be better to list them rather than saying !CLOSED and !STORED?

Reply:
Not sure will there be other status in the future. 
Happy to update it to OPEN for now.